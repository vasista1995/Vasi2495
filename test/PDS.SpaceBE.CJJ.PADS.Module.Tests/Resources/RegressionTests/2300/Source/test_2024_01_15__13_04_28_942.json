{
  "IdSource": "SPACEACT2:CJJ:BE:2Z3490JN:::Force_ShearOnBall:54077:243042504:0",
  "Data_flat_Metadata": {
    "Classification": "Capillary change",
    "Operator": "30156566",
    "Package": "CIPOS TINY",
    "WireSize": "30",
    "ParameterClass": "PPAP",
    "Segment": "IFPS",
    "Module": "Tiny",
    "Device": "IM393-M6F",
    "FourDReport": "*",
    "SpecialCharacteristics": "ISC",
    "ChipType": "IC",
    "WireType": "Gold",
    "Lot": "2Z3490JN",
    "ParameterName": "Force_ShearOnBall",
    "ParameterUnit": "g",
    "SiteKey": "CJJ",
    "ChannelId": "54077",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
    "CreatedTimestamp": "2023-12-11T23:07:46Z",
    "UpdatedTimestamp": "2023-12-11T23:07:46Z",
    "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
    "ExportedTimestampUtc": "2023-12-11T23:07:46Z",
    "ExportedTimestamp": "2023-12-11T23:07:46Z",
    "SampleTimestampUtc": "2023-12-11T14:07:46Z",
    "SampleTimestamp": "2023-12-11T23:07:46Z",
    "SourceDataLevel": "C",
    "ChannelName": "Ball_ShearForce/Tiny/Au/30",
    "ChannelDescr": "W/B",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/17/2023 19:54:51",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 602017,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2300",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 23.0,
        "SpecTarget": 50.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYYNYNNYYYYYYYYYYNNNYYYYYYNYYYYYYYY"
      },
      "SpaceAggregates": {
        "Mean": "52,844",
        "Median": "53,45",
        "Sigma": "1,60899968924795",
        "Min": "50,04",
        "Max": "53,99",
        "Q1": "53,01",
        "Q3": "53,73"
      },
      "ControlLimits": {
        "ctrl_mean_low": 43.4240449329405,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 51.6620883604506,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 59.9001317879608,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "ctrl_sigma_high": 11.6214694959517,
        "CtrlSigmaHighEnabled": "Y",
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_high": 30.033290763033,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 47.8057142857143,
        "ExtEwmaMV": 49.1004447532652,
        "ExtMsMV": 3.70099689273039,
        "EwmaS": 1.65445870429453,
        "EwmaR": 4.07134334376222,
        "ExtEwmaMVLCL": 47.5422450389591,
        "ExtEwmaMVCenter": 51.6625257886831,
        "ExtEwmaMVUCL": 55.7828065384071,
        "ExtMaMVLCL": 48.5478863042441,
        "ExtMaMVCenter": 51.6625257886831,
        "ExtMaMVUCL": 54.7771652731221,
        "EwmaSLCL": 1.59616058176069,
        "EwmaSCenter": 5.55900263797984,
        "EwmaSUCL": 9.52184469419899,
        "EwmaRLCL": 4.14654524083737,
        "EwmaRCenter": 13.7321758998435,
        "EwmaRUCL": 23.3178065588496,
        "ExtMSMVLCL": 0.00398304749839945,
        "ExtMSMVCenter": 1.87828561622881,
        "ExtMSMVUCL": 7.54514606020339,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 23.9820872788866,
        "RawUal": 79.3429642984796,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 243042504,
        "Seqnr": 3,
        "Value": 53.99,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243042504,
        "Seqnr": 2,
        "Value": 50.04,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243042504,
        "Seqnr": 1,
        "Value": 53.01,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243042504,
        "Seqnr": 5,
        "Value": 53.45,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 243042504,
        "Seqnr": 4,
        "Value": 53.73,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CJJ:BE:::Force_ShearOnBall:54077",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:2Z3490JN",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CJJ:BE:2Z3490JN:::Force_ShearOnBall:54077:243042504:0"
  }
}