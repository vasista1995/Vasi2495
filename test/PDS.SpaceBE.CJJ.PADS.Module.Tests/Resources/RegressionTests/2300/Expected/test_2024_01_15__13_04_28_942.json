{
  "Data_flat_Metadata": {
    "Classification": "Capillary change",
    "MeasLot": "2Z3490JN",
    "SiteKey": "CJJ",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-12-11T14:07:46Z",
    "BeginTimestamp": "2023-12-11T23:07:46Z",
    "EndTimestampUtc": "2023-12-11T14:07:46Z",
    "EndTimestamp": "2023-12-11T23:07:46Z"
  },
  "Data_flat_StrangeMetadata": {
    "WireSize": "30",
    "WireType": "Gold",
    "ChipType": "IC",
    "Operator": "30156566",
    "Package": "CIPOS TINY",
    "Device": "IM393-M6F"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 3,
          "SampleId": 243042504,
          "Value": 53.99,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-11T23:07:46Z",
          "SampleTimestampUtc": "2023-12-11T14:07:46Z",
          "CreatedTimestamp": "2023-12-11T23:07:46Z",
          "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
          "UpdatedTimestamp": "2023-12-11T23:07:46Z",
          "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
          "NumViolations": 0,
          "SampleMean": 52.844,
          "SampleMin": 50.04,
          "SampleMax": 53.99,
          "SampleMedian": 53.45,
          "SampleStdev": 1.60899968924795,
          "SampleQ1": 53.01,
          "SampleQ3": 53.73,
          "SampleSize": 5
        },
        {
          "Seqnr": 2,
          "SampleId": 243042504,
          "Value": 50.04,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-11T23:07:46Z",
          "SampleTimestampUtc": "2023-12-11T14:07:46Z",
          "CreatedTimestamp": "2023-12-11T23:07:46Z",
          "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
          "UpdatedTimestamp": "2023-12-11T23:07:46Z",
          "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
          "NumViolations": 0,
          "SampleMean": 52.844,
          "SampleMin": 50.04,
          "SampleMax": 53.99,
          "SampleMedian": 53.45,
          "SampleStdev": 1.60899968924795,
          "SampleQ1": 53.01,
          "SampleQ3": 53.73,
          "SampleSize": 5
        },
        {
          "Seqnr": 1,
          "SampleId": 243042504,
          "Value": 53.01,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-11T23:07:46Z",
          "SampleTimestampUtc": "2023-12-11T14:07:46Z",
          "CreatedTimestamp": "2023-12-11T23:07:46Z",
          "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
          "UpdatedTimestamp": "2023-12-11T23:07:46Z",
          "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
          "NumViolations": 0,
          "SampleMean": 52.844,
          "SampleMin": 50.04,
          "SampleMax": 53.99,
          "SampleMedian": 53.45,
          "SampleStdev": 1.60899968924795,
          "SampleQ1": 53.01,
          "SampleQ3": 53.73,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 243042504,
          "Value": 53.45,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-11T23:07:46Z",
          "SampleTimestampUtc": "2023-12-11T14:07:46Z",
          "CreatedTimestamp": "2023-12-11T23:07:46Z",
          "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
          "UpdatedTimestamp": "2023-12-11T23:07:46Z",
          "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
          "NumViolations": 0,
          "SampleMean": 52.844,
          "SampleMin": 50.04,
          "SampleMax": 53.99,
          "SampleMedian": 53.45,
          "SampleStdev": 1.60899968924795,
          "SampleQ1": 53.01,
          "SampleQ3": 53.73,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 243042504,
          "Value": 53.73,
          "is_flagged": "N",
          "SampleTimestamp": "2023-12-11T23:07:46Z",
          "SampleTimestampUtc": "2023-12-11T14:07:46Z",
          "CreatedTimestamp": "2023-12-11T23:07:46Z",
          "CreatedTimestampUtc": "2023-12-11T14:07:46Z",
          "UpdatedTimestamp": "2023-12-11T23:07:46Z",
          "UpdatedTimestampUtc": "2023-12-11T14:07:46Z",
          "NumViolations": 0,
          "SampleMean": 52.844,
          "SampleMin": 50.04,
          "SampleMax": 53.99,
          "SampleMedian": 53.45,
          "SampleStdev": 1.60899968924795,
          "SampleQ1": 53.01,
          "SampleQ3": 53.73,
          "SampleSize": 5
        }
      ],
      "ParameterClass": "PPAP",
      "Segment": "IFPS",
      "Module": "Tiny",
      "FourDReport": "*",
      "SpecialCharacteristics": "ISC",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 23.0,
          "target": 50.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 43.4240449329405,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 51.6620883604506,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 59.9001317879608,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "ctrl_stdev_high": 11.6214694959517,
          "CtrlStdevHighEnabled": "Y",
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_high": 30.033290763033,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 52.84400000000001,
        "median": 53.45,
        "stdev": 1.439133072373782,
        "range": 3.950000000000003,
        "min": 50.04,
        "max": 53.99,
        "q02": 50.2776,
        "q05": 50.634,
        "q25": 53.01,
        "q75": 53.73,
        "q95": 53.938,
        "q98": 53.9692,
        "NumViolations": 0,
        "Samples": "243042504"
      },
      "ParameterName": "Force_ShearOnBall",
      "ParameterUnit": "g",
      "ChannelId": "54077",
      "SourceDataLevel": "C",
      "channel_name": "Ball_ShearForce Tiny Au 30",
      "channel_description": "W B",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-12-11T14:07:46Z",
      "created_timestamp": "2023-12-11T23:07:46Z",
      "updated_timestamp_utc": "2023-12-11T14:07:46Z",
      "updated_timestamp": "2023-12-11T23:07:46Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-12-11T14:07:46Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:CJJ:BE:::",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:2Z3490JN",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "CJJ:BE:2Z3490JN:::Force_ShearOnBall:54077:243042504:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-12-11T23:07:46Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:2Z3490JN:SPACEAGGED2:CJJ:BE::::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "CJJ"
  }
}